packages/ia-components/components/track-lists/track-list.jsx

```diff
@@ -11,16 +11,13 @@ import OneTrack from './track';
  */
 const TheatreTrackList = (props) => {
   const {
-    selectedTrack, onSelected, tracks, displayTrackNumbers, creator: albumCreator, albumName
+    selectedTrack = 0, onSelected, tracks, displayTrackNumbers, creator: albumCreator, albumName
   } = props;
 
   if (!tracks.length) return <p className="no-tracks">No tracks to display.</p>;
 
-  const [firstTrack = {}] = tracks;
-  const { trackNumber: firstTrackNumber } = firstTrack;
-  const trackNumberToHighlight = Number.isInteger(selectedTrack)
-    ? selectedTrack
-    : firstTrackNumber;
+  const trackNumberToHighlight = selectedTrack === null ? 0 : selectedTrack;
```

Comment:
SUGGESITON: I think this could be more legible as:
```const trackNumberToHighlight = selectedTrack ?? 0;```

Reply:
```suggestion
  const trackNumberToHighlight = selectedTrack ?? 0;
```